superset-frontend/src/explore/controlPanels/FilterBox.jsx

```diff
@@ -55,10 +55,10 @@ export default {
               type: 'CheckboxControl',
               label: t('Instant Filtering'),
               renderTrigger: true,
-              default: true,
-              description:
-                'Whether to apply filters as they change, or wait for ' +
-                'users to hit an [Apply] button',
+              default: false,
+              description: t(
+                'Whether to apply filters as they change, or wait for users to hit an [Apply] button',
```

Comment:
Maybe we should add a better description while we're in here too. @sylvia-tomiyama any ideas?

Reply:
I checked what other descriptions look like (for consistency...). How about _Check to apply filters instantly as they change instead of displaying [Apply] button._